server/Server.cpp

```diff
@@ -91,20 +112,26 @@ void Server::CheckForConnections() {
 		it = Fds.begin() + 1;
 		mt = connectionMsgs.begin();
 		while (it != Fds.end()) {
-			if ((it->revents & POLLIN) != 0){
+			if ((it->revents & POLLERR) != 0) {
+				std::cout << "Socket error Occurred" << std::endl;
+				cleanConnection();
```

Comment:
do we need to set the response http status code to `Internal server error` when errors in ssocket or send/recieve occur, or does it not make sense?